react-ui/src/components/ListItemButton.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * Displaying row with external data in vertically stacked list
```

Comment:
I'm not sure that external data is that clear. The data itself is internal, clicking on the button leads to an external site.

Reply:
Edited